admin/modules/config/banning.php

```diff
@@ -35,6 +35,29 @@
 		$errors[] = $lang->error_filter_already_banned;
 	}
 
+	if($mybb->input['type'] == 1)
```

Comment:
Suggestion: wrap this conditional in an outer conditional `if(!$errors)`.

Reply:
I added `!$errors` into the same condition, as I think it's a little bit more compact. Do you agree? :)

Reply:
Yes, that works fine.